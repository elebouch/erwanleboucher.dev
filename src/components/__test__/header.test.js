import React from "react"
import { render } from "@testing-library/react"
import "jest-styled-components"

import Header from "../header"

describe("Header", () => {
  it("renders correctly", () => {
    const { container } = render(<Header siteTitle="Default Starter" />)
    expect(container.innerHTML).toMatchSnapshot()
  })
})
