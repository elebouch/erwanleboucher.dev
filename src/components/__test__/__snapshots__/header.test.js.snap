// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header>
  <div
    className="sc-AxjAm ikxNPC"
  >
    <a
      className="sc-AxirZ dribLd sc-AxhCb gCMPnZ"
      href="/"
    >
      Default Starter
    </a>
    <div
      className="sc-AxiKw elWQB"
    >
      <a
        className="sc-AxhCb gCMPnZ"
        href="/"
      >
        About
      </a>
      <a
        className="sc-AxhCb gCMPnZ"
        href="/journey"
      >
        Journey
      </a>
      <a
        className="sc-AxhCb gCMPnZ"
        href="/projects"
      >
        Projects
      </a>
    </div>
  </div>
</header>
`;
